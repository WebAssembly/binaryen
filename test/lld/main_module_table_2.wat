(module
 (import "env" "__stack_pointer" (global $sp (mut i32)))
 (import "GOT.func" "__stdio_write" (global $gimport$9 (mut i32)))
 (import "env" "table" (table $timport$9 1 funcref))
 (global $global i32 (i32.const 42))
 (export "__stdio_write" (func $__stdio_write))
 (export "__data_end" (global $global))
 (func $__stdio_write
 )
)
