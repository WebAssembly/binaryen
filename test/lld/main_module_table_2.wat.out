(module
 (type $none_=>_none (func))
 (type $none_=>_i32 (func (result i32)))
 (import "env" "table" (table $timport$9 1 funcref))
 (import "env" "__stack_pointer" (global $sp (mut i32)))
 (import "env" "fp$__stdio_write$v" (func $fp$__stdio_write$v (result i32)))
 (global $gimport$9 (mut i32) (i32.const 0))
 (global $global i32 (i32.const 42))
 (export "__stdio_write" (func $__stdio_write))
 (export "__data_end" (global $global))
 (export "__assign_got_enties" (func $__assign_got_enties))
 (func $__stdio_write
  (nop)
 )
 (func $__assign_got_enties
  (global.set $gimport$9
   (call $fp$__stdio_write$v)
  )
 )
)
(;
--BEGIN METADATA --
{
  "staticBump": 4294966770,
  "tableSize": 1,
  "initializers": [
    "__assign_got_enties"
  ],
  "declares": [
    "fp$__stdio_write$v"
  ],
  "externs": [
    "___stack_pointer"
  ],
  "exports": [
    "__stdio_write",
    "__assign_got_enties"
  ],
  "namedGlobals": {
    "__data_end" : "42"
  },
  "invokeFuncs": [
  ],
  "mainReadsParams": 0,
  "features": [
  ]
}
-- END METADATA --
;)
